packages/qunit-dom/lib/assertions/exists.ts

```diff
@@ -1,4 +1,4 @@
-import { ExistsOptions } from '../assertions';
+import type { ExistsOptions } from '../assertions.js';
```

Comment:
the need for type here is enforced from `verbatimModuleSyntax` -- more information in the tsconfig.json